src/main/kotlin/eu/europa/ec/eudi/sdjwt/SdJwtPresent.kt

```diff
@@ -85,4 +85,4 @@ interface SdJwtPresentationOps<JWT> : SdJwtRecreateClaimsOps<JWT> {
 /**
  * Creates a Presentation that discloses **ALL** the claims of this Issuance.
  */
-fun <JWT> SdJwt.Issuance<JWT>.present(): SdJwt.Presentation<JWT> = SdJwt.Presentation(jwt, disclosures)
+fun <JWT> SdJwt<JWT>.present(): SdJwt<JWT> = SdJwt(jwt, disclosures)
```

Comment:
@babisRoutis This method can be safely removed.

Reply:
@dzarras 
There could be an issue, if we remove this and it is related on how we have implemented the 
`fun present(query: Set<ClaimPath>): SdJwt<JWT>?` method.

If you pass an empty set to this method, then you get always a `null` result.

On the other hand the OpenId4VP DCQL defines the following semantics:
* Verifier may define a set of claim (paths), or even some alternative sets of claim (paths), to be presented.
* In case a Verifier doesn't provide such paths, then it is implied that the query is about all claims

So, to remove the method that you mention, IMHO, we need to change the behavior of the other method when passing an empty set.

What do you think?

Reply:
Removed the method.
Redefined the `present(query)` variant. In case of an empty query, returns the input SD-JWT

Reply:
@babisRoutis Agreed.

I propose the following:

`fun present(query: Set<ClaimPath>): Result<SdJwt<JWT>>`
* if query is empty, return SdJwt as is
* if query is non-empty:
  * if it contains irrelevant claim paths, fail
  * otherwise, create a new sd-jwt with the appropriate disclosures